code/site/libraries/plugin.php

```diff
@@ -610,4 +616,17 @@ public function getUser()
 	{
 		return $this->user;
 	}
+	
+	/**
+	 * Get installed API version
+	 * 
+	 * @return string
+	 */
+	
+	public function getInstalledAPIVersion()

```

Comment:
@twsvaishali I have removed it from the http.php file and called it from plugin.php